internal/pkg/cli/pipeline_init.go

```diff
@@ -645,7 +646,9 @@ func (o *initPipelineOpts) createPipelineManifest() error {
 
 	manifestMsgFmt := "Wrote the pipeline manifest for %s at '%s'\n"
 	if manifestExists {
-		manifestMsgFmt = "Pipeline manifest file for %s already exists at %s, skipping writing it.\n"
+		manifestMsgFmt = `Pipeline manifest file for %s already exists at %s, skipping writing it.
```

Comment:
If they already exist, can we make these a `log.Debug` instead of `log.Success` (samesies for buildspec)

Reply:
The next line is a `log.Debug` already. I feel like this information will get lost with the mass of grey. How do you feel about `log.Info`?

Reply:
Info sounds good! I just want to increase our Signal/Noise ratio, maybe we can do `Info` and remove the second sentence from the logs ("Previously set repository URL, branch, and environment stages will remain." , "Previously set config will remain.")

Reply:
Oh, I added those because I want it to be clearer that even though we're (still) asking for envs, if they chose different ones than those in the existing manifest, the change won't be applied.

Reply:
Oh I wanted to keep `Successf` when we write the file but `Infof` if it already exists :rip:

Reply:
🤦🏼  done